akka-http-testkit/src/main/scala/akka/http/scaladsl/testkit/RouteTestTimeout.scala

```diff
@@ -7,9 +7,14 @@ package akka.http.scaladsl.testkit
 import scala.concurrent.duration._
 import akka.actor.ActorSystem
 import akka.testkit._
+import akka.http.impl.util.enhanceConfig
+import com.typesafe.config.{Config, ConfigFactory}
 
 case class RouteTestTimeout(duration: FiniteDuration)
 
 object RouteTestTimeout {
-  implicit def default(implicit system: ActorSystem): RouteTestTimeout = RouteTestTimeout(1.second.dilated)
+
+  private val testConfig: Config = ConfigFactory.load()
```

Comment:
This means it can only be set in the default config, read once per JVM and not possible to override for individual tests. 

Instead it should come from the actor system config, something like this: `system.settings.config.getFiniteDuration("akka.http.testkit.routes.timeout")`. We should probably also keep dilating it to not break existing tests depending on a scaled 1 second

Reply:
@johanandren I fixed this point, with getting the config from the actor system settings. Let me know if now it sounds ok.

Reply:
This looks better, let's see if ci is happy!